src/Microsoft.ML.CpuMath/SseIntrinsics.cs

```diff
@@ -406,44 +406,44 @@ internal static unsafe void MatMulTranA(bool add, AlignedArray mat, AlignedArray
         }
 
         // dst[i] += scale
-        internal static unsafe void AddScalarU(float scale, Span<float> dst)
+        public static unsafe void AddScalarU(float scalar, Span<float> dst)
         {
             fixed (float* pdst = dst)
             {
                 float* pDstEnd = pdst + dst.Length;
                 float* pDstCurrent = pdst;
 
-                Vector128<float> x1 = Sse.SetAllVector128(scale);
+                Vector128<float> scalarVector = Sse.SetAllVector128(scalar);
```

Comment:
Typically, changes like this should go in a separate PR.  We shouldn't be adding AVX support, and then doing unrelated cleanup/refactoring of SSE code in the same change.

Reply:
Sorry my bad... I'll keep this in mind.